pkg/composableschemadsl/parser/tests/useafterdef.zed.expected

```diff
@@ -8,9 +8,14 @@ NodeTypeFile
       end-rune = 21
       input-source = use after definition
       start-rune = 0
-    NodeTypeError
-      end-rune = 22
-      error-message = Unexpected token at root level: TokenTypeKeyword
-      error-source = use
+    NodeTypeUseFlag
+      end-rune = 37
       input-source = use after definition
-      start-rune = 24
\ No newline at end of file
+      start-rune = 24
+      use-flag-name = expiration
+      child-node =>
+        NodeTypeError
+          end-rune = 37
+          error-message = `use` expressions must be declared before any definition
+          input-source = use after definition
+          start-rune = 38
```

Comment:
Adding more context to this parsing error.

Is it an issue that the start rune is after the end rune?